pkg/operator/ceph/object/spec.go

```diff
@@ -280,17 +280,39 @@ func (c *clusterConfig) makeRGWPodSpec(rgwConfig *rgwConfig) (v1.PodTemplateSpec
 }
 
 func (c *clusterConfig) createCaBundleUpdateInitContainer(rgwConfig *rgwConfig) v1.Container {
-	caBundleMount := v1.VolumeMount{Name: caBundleVolumeName, MountPath: caBundleSourceCustomDir, ReadOnly: true}
+	caBundleMount := v1.VolumeMount{Name: caBundleVolumeName, MountPath: "/etc/ssl/custom-ca", ReadOnly: true}
 	volumeMounts := append(controller.DaemonVolumeMounts(c.DataPathMap, rgwConfig.ResourceName, c.clusterSpec.DataDirHostPath), caBundleMount)
 	updatedCaBundleDir := "/tmp/new-ca-bundle/"
 	updatedBundleMount := v1.VolumeMount{Name: caBundleUpdatedVolumeName, MountPath: updatedCaBundleDir, ReadOnly: false}
 	volumeMounts = append(volumeMounts, updatedBundleMount)
 	return v1.Container{
 		Name:    "update-ca-bundle-initcontainer",
-		Command: []string{"/bin/bash", "-c"},
-		// copy all content of caBundleExtractedDir to avoid directory mount itself
+		Command: []string{"/bin/bash", "-c", "-x", "-e"},
 		Args: []string{
-			fmt.Sprintf("/usr/bin/update-ca-trust extract; cp -rf %s/* %s", caBundleExtractedDir, updatedCaBundleDir),
+			fmt.Sprintf(`
```

Comment:
I have a general concern about shell scripts over a few lines being inlined.  My experience is that they often have simple errors in them or become broken by accident over time. The way I have partially mitigated this is to write out non-trivial fragments as a `foo.sh` file, which is then at least covered by shellcheck under CI, and is easier to execute by hand in development mode.

Reply:
I second this